bin/person.rb

```diff
@@ -1,4 +1,12 @@
+class Corrector
+  def correct_name(name)
+    name.length > 10 ? name = name[0..10] : name
+    name.capitalize!
+  end
+end
```

Comment:
- [ ] Please put this class in a separate .rb file and then require it in the person.rb file. It is better for each class to have its own file.